src/main/java/frc/robot/subsystems/drive/PhoenixOdometryThread.java

```diff
@@ -100,7 +100,7 @@ public void run() {
       signalsLock.lock();
       try {
         if (isCANFD) {
-          BaseStatusSignal.waitForAll(2.0 / DriveConstants.Module.ODOMETRY_FREQUENCY, signals);
+          BaseStatusSignal.waitForAll(4.0 / DriveConstants.Module.ODOMETRY_FREQUENCY, signals);
```

Comment:
This is potentially a really good fix. Can we try setting it to 1.5 or 1.0 in the lab and see if the issue (last CAN ID of wire is not getting enough time during init sequence, causing timeout) manifests itself more reliably to make sure we found the root cause? Then we can tell drive team they don't need to keep twisting the wheels on the field.